packages/documentation/copy/en/declaration-files/Publishing.md

```diff
@@ -33,7 +33,7 @@ For example:
 
 Note that the `"typings"` field is synonymous with `types`, and could be used as well.
 
-Also note that if your main declaration file is named `index.d.ts` and lives at the root of the package (next to `index.js`) you do not need to mark the `types` property, though it is advisable to do so.
+Also note that if your main declaration file is named `index.d.ts` and lives at the root of the package (next to the `package.json`) you do not need to mark the `types` property, though it is advisable to do so.
```

Comment:
I think this change is more confusing, the `index.d.ts` would be found relative to the field defined in `main` which is being described as `index.js` here - not relative to the `package.json`.  Perhaps:

```suggestion
Note:  TypeScript will infer the root `*.d.ts` based on the value in `main`, which means in cases like above you do not need to mark the `types` property, though it is advisable to do so.
```

Reply:
Not only was my change confusing, it was wrong. ;) it was also how I assumed it's working from reading the docs.
Thanks for the suggestion and making it clear for me, and hopefully others.

Why `*.d.ts`? Is there a list of files that are searched for, @orta?

Reply:
The filename is dynamic because it could be anything in the `"main"` - the * could also be `[file].d.ts` and I think it would also make sense

Reply:
The filename is dynamic because it could be anything in the `"main"` - the * could also be `[file].d.ts` and I think it would also make sense